apps/system/style/quick_settings/quick_settings.css

```diff
@@ -54,6 +54,11 @@
   opacity: 0.5;
 }
 
+#quick-settings-airplane-mode[data-disabling] {
+  color: #ffffff;
```

Comment:
per gaia system wide we prefer `#fff` instead of `#ffffff`, but both of them are valid CSS though.